.github/workflows/nightly-wheel.yml

```diff
@@ -56,6 +53,25 @@ jobs:
               ~/.local/bin/aws s3 cp "$pkg" "$S3_PATH" --acl public-read
           done
 
+      - name: Install the freshly uploaded TorchArrow
+        run: |
+          pip install --pre torcharrow -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
curious: does this guarantee to download the just uploaded TA nightly? ;) 

cc @ejguan , @NivekT

Reply:
Probably not. I believe it will take a few minutes for the just uploaded wheels becoming available from the link. Why not just install from your local wheel?

Reply:
yep that's probably better, lemme tweak it that way.